plugins/base/src/main/resources/dokka/templates/includes/article.ftl

```diff
@@ -0,0 +1,3 @@
+<#macro display>
+        <@content/>
+</#macro>
```

Comment:
Maybe rename the file it to `page_content`, `main_content` or just `content`? 

I think `article` is also 50/50, you think you know what it is, but still have to check it, whereas something with `content` would be 100% clear to me